db/migrate/20170725143941_create_api_consumers.rb

```diff
@@ -0,0 +1,12 @@
+class CreateApiConsumers < ActiveRecord::Migration
+  def change
+    create_table :api_consumers do |t|
+      t.string :name
+      t.string :email, null: false, index: true
+      t.string :provider, null: false
+      t.string :uid, null: false, index: true
```

Comment:
@andersoncardoso @coffeejunk I think the uid shoud be a autoincrement or something like that. we want to allow to extensibility, so currently implemeting OpenIDC but we would like to allow the user to authenticate in other form, so uid should be agnostic to the way the user authenticates.